src/runtime_src/core/common/api/xrt_module.cpp

```diff
@@ -408,6 +408,11 @@ class module_impl
 // construct patcher objects for each argument.
 class module_elf : public module_impl
 {
+  // rela->addend have offset to base-bo-addr info along with schema
+  // [0:3] bit are used for patching schema, [4:31] used for base-bo-addr
+  constexpr static uint32_t Addend_Shift = 4;
+  constexpr static uint32_t Addend_Mask = ~((uint32_t)0) << Addend_Shift;
+  constexpr static uint32_t Schema_Mask = ~Addend_Mask;
```

Comment:
The coding style for better or worse in `core/common` is no `Camel_Case`, so could you change to `no_camel_case`  :-)

Reply:
Please refer to line 678: const ELFIO::Elf32_Rela
The code style is already there.
Please...

Reply:
There are more:

      auto begin = reinterpret_cast<const ELFIO::Elf32_Rela*>(sec->get_data());
      auto end = begin + sec->get_size() / sizeof(const ELFIO::**Elf32_Rela**);
      for (auto rela = begin; rela != end; ++rela) {
        auto symidx = ELFIO::get_sym_and_type<ELFIO::Elf32_Rela>::get_r_sym(rela->r_info);

        auto dynsym_offset = symidx * sizeof(ELFIO::**Elf32_Sym**);

Reply:
We can hardly do anything about code that is not our code.
Please just follow the existing coding style of the code you edit.  And it seems like the 3 lines I pointed to are indeed our code lines.